pkg/kubectl/polymorphichelpers/canbeautoscaled.go

```diff
@@ -19,16 +19,35 @@ package polymorphichelpers
 import (
 	"fmt"
 
+	apps "k8s.io/api/apps/v1"
```

Comment:
nit: mind renaming to `appsv1`, `corev1`, `extensionsv1beta1` to keep consistency with other files? (e.g. https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/polymorphichelpers/protocolsforobject.go#L23-L27 or https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/polymorphichelpers/updatepodspec.go#L22)